data/discovery/field/citizens-advice-bureau-contact-region.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: citizens-advice-bureau-contact-region
+phase: discovery
+register: citizens-advice-bureau-contact-region
+text: Contact region for Citizens Advice Bureaux
```

Comment:
`Contact region of a Citizens Advice Bureau in England or Wales`.

Here and elsewhere I'm assuming that this is a register of bureaux in England and Wales, because that's what's in the data.  But [Wikipedia](https://en.wikipedia.org/wiki/Citizens_Advice#Organisation) says that bureaux in Northern Ireland are part of two different organisations.

> All bureaux in England, Wales and Northern Ireland are members of Citizens Advice, the operating name of The National Association of Citizens Advice Bureaux. Northern Ireland bureaux are also members of the Northern Ireland Association of Citizens Advice Bureaux (NIACAB). Bureaux in Scotland are members of Citizens Advice Scotland (CAS), part of the Scottish Association of Citizens Advice Bureaux.

Something to note in the README of the data repo and to discuss with the custodian.